### ISSUE TITLE
get_video_info broken

### ISSUE THREAD

**Issue by 89z** [benkaiser/Stretto-Helper-Extension#1]
~~~
curl -v 'https://www.youtube.com/get_video_info?c=TVHTML5&cver=7.20190319&eurl=https://youtube.googleapis.com/v/fEqAsHTd1BQ&gl=US&hl=en&html5=1&ps=default&video_id=fEqAsHTd1BQ'
HTTP/2 404
~~~

https://github.com/benkaiser/Stretto-Helper-Extension/blob/87db994e2095303e2a1ff8b1fe86a19858fd42ac/extension/background.js#L3542-L3554

**Comment by benkaiser** [comment]
Hey @89z , I managed to root-cause this tonight.

Looks like we needed a few fixes in place to get it working. You should be able to update the extension now and it'll work again with the latest versions of Chrome.

Let me know if it's still bugging out on you

**Comment by 89z** [comment]
I dont think so, still seems to be broken:

~~~
curl -v 'https://www.youtube.com/get_video_info?c=TVHTML5&cver=7.20210622.10.00&eurl=https://youtube.googleapis.com/v/7GIuUNaq5PI&gl=US&hl=en&html5=1&ps=default&video_id=7GIuUNaq5PI'
HTTP/2 404
~~~

**Comment by benkaiser** [comment]
Did you reinstall the extension?

**Comment by 89z** [comment]
sorry, what I mean to say, is that it appears now that anything with `/get_video_info` is dead code. So unless someone figures a way to get that endpoint working again, it might make sense to just remove calls to that endpoint

**Comment by benkaiser** [comment]
I'm not sure if you're aware, but this extension contains a ytdl version
that is a fork of a fork.

Ytdl-core is where the main development happens, you might be better off
placing an issue there if you've got a gripe with the ytdl code.

React-native-ytdl is what I forked off because I need the removed
dependencies so it works in the browser, but then they added some very
react-native specific things which broke in the web, so I forked them and
removed those small changes.
